python/pfs/instmodel/lightSource.py

```diff
@@ -211,9 +211,12 @@ def getTargetData(self, fiberId):
         patch = self.pfsDesign.patch[index]
         targetType = self.pfsDesign.targetType[index]
         fiberStatus = self.pfsDesign.fiberStatus[index]
-        fiberMags = dict(zip(self.pfsDesign.filterNames[index], self.pfsDesign.fiberMag[index]))
-        return SimpleNamespace(index=index, catId=catId, objId=objId, tract=tract, patch=patch,
-                               targetType=targetType, fiberStatus=fiberStatus, fiberMags=fiberMags)
+        fiberFlux = dict(zip(self.pfsDesign.filterNames[index],
+                         self.pfsDesign.fiberFlux[index]))
```

Comment:
Needs extra indentation.

Reply:
Which line requires the extra indentation? It passes as-is with my flake8 checker on VSCode. For line 215, any extra spaces (or fewer) results in a E128 error for me.